lib/components/molecules/Popover/Popover.style.js

```diff
@@ -0,0 +1,18 @@
+import styled from 'styled-components';
+
+const PopoverContentWrapper = styled.div`
```

Comment:
This is not the right way, just encapsulate all CSS in a class and apply that to whole component. Its easier that way to make variations.